CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 
 # Thanks for contributing!
+:+1: First of all, thanks for contributing! The team is happy to help if you have any questions. Have a look to this contributing guide and also to the [FAQ section](https://github.com/KnpLabs/snappy/blob/master/doc/faq.md). :feet: And also to our [Code of Conduct](https://github.com/KnpLabs/snappy/blob/master/CODE_OF_CONDUCT.md)
 
+The following is a set of guidelines for contributing to Snappy, which is hosted in the [KNP Labs Organization](https://github.com/KnpLabs) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request. :shipit:
```

Comment:
What do you think about putting the code of conduct instructions in a different chapter?

```suggestion
:+1: First of all, thanks for contributing! The team is happy to help if you have any questions. Have a look to this contributing guide and also to the [FAQ section](https://github.com/KnpLabs/snappy/blob/master/doc/faq.md). :feet:

The following is a set of guidelines for contributing to Snappy, which is hosted by the [KNP Labs Organization](https://github.com/KnpLabs) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document opening a pull request. :shipit:

## Code of Conduct
This project and everyone participating in it is governed by the following [Code of Conduct](https://github.com/KnpLabs/snappy/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
```